src/java.security.jgss/share/classes/sun/security/jgss/wrapper/GSSNameElement.java

```diff
@@ -48,11 +49,12 @@
  */
 
 public class GSSNameElement implements GSSNameSpi {
+    private final Cleaner.Cleanable cleanable;
 
     long pName = 0; // Pointer to the gss_name_t structure
```

Comment:
Did you mean pName? The dispose() method will reset it to zero.  'pName" is used a lot in native implementation.  It may be doable to make it final, but it may be more complicated than I could expect.  I would like to leave it as it is in this PR.

Reply:
IMO, there's no need to reset it to zero in `dispose()`. As for the usage in native implementation, if there is really a case that it gets updated, then the cleanable you registered at the beginning will be useless. Isn't that a real problem?

Reply:
I don't think anything is guaranteed after a name is disposed. I see no problem in doing this.